targets/targets.json

```diff
@@ -564,6 +564,24 @@
         "release_versions": ["2", "5"],
         "device_name": "MKW41Z512xxx4"
     },
+    "MCU_K24F1M": {
+        "core": "Cortex-M4F",
+        "supported_toolchains": ["ARM", "GCC_ARM", "IAR"],
+        "extra_labels": ["Freescale", "MCUXpresso_MCUS", "KSDK2_MCUS", "MCU_K24F", "MCU_K24F1M", "KPSDK_MCUS", "KPSDK_CODE", "FLASH_CMSIS_ALGO"],
```

Comment:
Same comment on this line: you don't need to add yourself to extra_labels, and you should not.